src/elexmodel/distributions/GaussianModel.py

```diff
@@ -70,7 +72,7 @@ def _get_n_units_per_group(self, conformalization_data, nonreporting_units, aggr
             .fillna({"n": 0})
         )
 
-    def _fit(self, conformalization_data, estimand, aggregate, alpha, beta):
+    def _fit(self, conformalization_data, estimand, aggregate, alpha, beta, winsorize):
```

Comment:
I just realize we need don't need to pass `beta` or `winsorize` into `fit` or `_fit` because they are passed into the `GaussianModel` constructor as part of `model_settings`, right?